network_wrangler/RoadwayNetwork.py

```diff
@@ -743,27 +760,26 @@ def apply_roadway_feature_change(self, properties: dict) -> RoadwayNetwork:
         # shallow (copy.copy(self)) doesn't work as it will still use the references to links_df etc from the original net
         updated_network = copy.deepcopy(self)
 
-        for attribute, values in properties.items():
-            existing_value = None
-
-            if "existing" in values.keys():
-                existing_value = values["existing"]
+        for p in properties:
+            attribute = p["property"]
 
-                # if existing value in project card is not same in the network
+            # if project card specifies an existing value in the network
+            #   check and see if the existing value in the network matches
+            if p.get("existing"):
                 network_values = updated_network.links_df[
                     updated_network.links_df["selected_links"] == 1
                 ][attribute].tolist()
-                if not set(network_values).issubset([existing_value]):
+                if not set(network_values).issubset([p.get("existing")]):
                     WranglerLogger.warning(
                         "WARNING: Existing value defined for {} in project card does not match the value in the roadway network for the selected links".format(
                             attribute
                         )
                     )
 
-            if "set" in values.keys():
-                build_value = values["set"]
+            if "set" in p.keys():
+                build_value = p["set"]
             else:
-                build_value = values["existing"] + values["change"]
+                build_value = p["existing"] + p["change"]
```

Comment:
question: i think we mean for this to be the existing on the network, not the existing in the project card.

Reply:
yes, if we are just warning (and not throwing error) when the existing in the project card is not same as existing on the network.

Reply:
done in c4e7fc98ad2a8e01431519c64e50623610308386